### PR TITLE
Helpflag added in main.dart

### PR DESCRIPTION
**Pull request by asaravindkumar**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -29,6 +29,19 @@ void main(List<String> args) async {
   var path = '';
   var projectName = '';
   var designType = 'sketch';
+  var configurationPath;
```
Comment (ivan-015) [inline lib/main.dart]: Hi @asaravindkumar , this line 32 causes an error with line 45 since configurationPath is declared again

Comment (asaravindkumar) [reply lib/main.dart]: @ivan-015  apologies. I removed the variable creation and committed it

**Comment by ivan-015** [comment]
Thanks for PR @asaravindkumar , I like the information you added! The only problem I see is that if you run `parabeac.dart` without arguments or with the `-h` option it won't be forwarded to `main.dart`, so we will only ever see help dialogue for `parabeac.dart`. My suggestion is that the help dialogue for `parabeac.dart` is printed and we can call `main.dart` regardless so that we get both help dialogues.

File: lib/main.dart (hunk 1 above)
Comment (ivan-015) [reply lib/main.dart]: Thank you!

**Comment by asaravindkumar** [comment]
@ivan-015  I removed the return and also wrapped the switch-case inside else of -h check. hope this serves the request.

**Review by ivan-015 — APPROVED** [review]
Looks good to me! Thanks for the PR

**Review by x64Eddie — APPROVED** [review]
LGTM! 🎊

**Pull request merged**